src/mono/mono/mini/exceptions-amd64.c

```diff
@@ -617,6 +649,75 @@ mono_arch_get_throw_corlib_exception (MonoTrampInfo **info, gboolean aot)
 }
 #endif /* !DISABLE_JIT */
 
+#if defined(AMD64_CALLEE_SAVED_XREGS) && defined(TARGET_WIN32)
+
+static gboolean
+unwind_llvm_frame_win64 (
+	gpointer ip,
+	mono_unwind_reg_t *regs,
+	int nregs,
+	gboolean readonly_regs,
+	host_mgreg_t **save_locations,
+	int save_locations_len)
+{
+	gboolean result = FALSE;
```

Comment:
Would be better to either fix llvm to emit dwarf unwind info for these, or add this unwind info to the generic info when thats loaded.

Reply:
I think we should move back to just use windows unwind ops for llvm frames, right now we enable dwarf + native unwind codes in llvm generated code, and then dwarf is not following the windows ABI (since it misses the XMM registers) since the dwarf implementation was not meant to be used under Windows. It is probably better to add handling of additional native unwind codes for llvm frames and remove the patch that emits dwarf for llvm frames. That will also remove the duplication that we currently carry in the Windows x64 images. That change could be done as a follow up to this PR, if needed.